src/commands/registers.ts

```diff
@@ -0,0 +1,56 @@
+import * as vscode from "vscode";
+import { EmacsCommand, IEmacsCommandInterrupted } from ".";
+
+// Will bind this this to C-x r s
+export class StartRegisterSaveCommand extends EmacsCommand implements IEmacsCommandInterrupted {
+  public readonly id = "StartRegisterSaveCommand";
+
+  private acceptingRegisterSaveCommand = false;
+
+  public startRegisterSaveCommand(): void {
```

Comment:
Can you make this private?

Reply:
Sorry this was originally my bad.
Fixed the original code you referred to as the basis: https://github.com/whitphx/vscode-emacs-mcx/pull/1680